nixos/modules/services/desktops/pipewire/README.md

```diff
@@ -1,6 +1,6 @@
 # Updating
 
 1. Update the version & hash in pkgs/development/libraries/pipewire/default.nix
-2. run `nix build -f /path/to/nixpkgs/checkout pipewire pipewire.mediaSession`
```

Comment:
This was probably an accidental deletion? The mediaSession output contains the config templates for the session manager.

Reply:
The `mediaSession` output is built with just `nix build -f /path/to/nixpkgs pipewire`:
```
$ rm result*
$ nix build -f . pipewire
$ echo result*
result result-dev result-doc result-installedTests result-jack result-lib result-mediaSession result-pulse
```

Reply:
I only get `result` if I do that. Might be down to some nix option in configuration.nix?